swagger.yml

```diff
@@ -74,6 +74,9 @@ paths:
               $ref: '#/definitions/site'
         default:
           $ref: '#/responses/error'
+      description: >-
+        Environment variable keys and values inside `build_settings.env` and `repo.env` will soon be removed. 
```

Comment:
I wonder if this message might make some folks panic that their env vars will be completely removed from the platform? I wonder if we might prevent such a misread, e.g.

> Environment variable keys and values will soon be moved from `build_settings.env` and `repo.env` to a new service.